dev/archery/archery/integration/runner.py

```diff
@@ -430,6 +430,11 @@ def run_all_tests(with_cpp=True, with_java=True, with_js=True,
             "middleware",
             description="Ensure headers are propagated via middleware.",
         ),
+        Scenario(
+            "ordered",
+            description="Ensure FlightInfo.ordered is supported.",
+            skip={"Java", "JS", "C#", "Go", "Rust"},
```

Comment:
I can put together Java support too.

Reply:
Wow! Thanks!
Could you push it to this branch when you work on it? Or you can create another branch for it.

Reply:
https://github.com/kou/arrow/pull/11

Reply:
Thanks!
Merged.